### DISCUSSION TITLE
Simulation modes DNS, LES, VLES, SVLES

### DISCUSSION THREAD

**Discussion by gio309246** [firemodels/fds#13661] [Q&A]
Hello everyone,

I’m currently in a phase of study and design, and I would need your help to clarify some questions regarding the simulation modes DNS, LES, VLES, and SVLES. I’m a beginner with Pyrosim and, despite having read the user guide, I unfortunately haven’t been able to fully understand the differences between these simulation mode and how they work.

In particular, I’m interested in understanding which of these modes is most suitable for my project, which involves modeling an high building. It’s crucial for me to use a fine mesh, on the order of centimeters (5 or 10 centimeters), to accurately capture the stratigraphy of a ventilated facade.

Could anyone explain to me clearly and simply the characteristics of each simulation mode and suggest which one might be the most appropriate for my specific case?

Thank you in advance for your help!

**Comment by mcgratta** [comment]
The FDS Technical Reference Guide explains these in detail. Search on, say, `SVLES` and you'll see an explanation of all. As for your application, use the default, `VLES`.

**Reply by gio309246** [reply]
Thank you very much for your response! 
I’ve read the guide and done some research, but I didn't find what I was looking for. I’m not interested in the physical and fluid dynamics aspects; my goal is to understand how fires evolve and propagate in facades. I was wondering which simulation mode would be the most suitable for my case, considering that I have a very large object to study, but with thicknesses on the order of centimeters (for example, 5 cm - a layer of the stratigraphy). Could you please advise me?

**Reply by Vojta-Salek** [reply]
I feel like if your goal is to understand how fire evolves, there is no way to skip physical and fluid dynamics aspects because the behavior of fire is dependent on the mentioned phenomena. 

Table 19.1 should help you make your decision. Based on the scale of your problem, you can cross out DNS and then you may want to think about how `CFL_VELOCITY_NORM`, `CHECK_VN`, etc. setup influence your case. How does changing this parameter affect my simulation? Is it important with respect to the required accuracy and my objectives? What are the costs and benefits of changing the parameter?

![image](https://github.com/user-attachments/assets/62c32f06-3f8c-4b0e-af5a-fec5ec5b5c5c)

**Reply by gio309246** [reply]
Thank you for your response! I appreciate the advice: I will read the guide carefully and make my evaluations. If anyone has further insights or suggestions, they are very welcome!
